config/crd/bases/metal3.io_baremetalhosts.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.4.1
+    controller-gen.kubebuilder.io/version: v0.4.0
```

Comment:
Why a version downgrade?

Reply:
My guess is that the local controller-gen version is different. Could be remedied easily.